lib/components/src/blocks/Story.tsx

```diff
@@ -65,8 +66,13 @@ const InlineStory: FunctionComponent<InlineStoryProps> = ({ storyFn, height, id
   </div>
 );
 
-const IFrameStory: FunctionComponent<IFrameStoryProps> = ({ id, title, height = '500px' }) => (
-  <div style={{ width: '100%', height }}>
+const IFrameStory: FunctionComponent<IFrameStoryProps> = ({
+  id,
+  title,
+  height = '500px',
+  width = '950px',
```

Comment:
Can we make this completely optional?

Reply:
That's actually just a default height and width. In case the IFrameStory component does not receive those parameters, it would just set it to 950x500 px...

Could you explain a little bit more about how this could be optional?